monero-core.pro

```diff
@@ -86,7 +86,8 @@ linux {
         -lssl \
         -lcrypto \
         -Wl,-Bdynamic \
-        -lunwind \
+        # currently monero has an issue with "static" build and linunwind-dev
+        # -lunwind \
```

Comment:
added this option and created PR https://github.com/monero-project/monero-core/pull/30